.circleci/config.yml

```diff
@@ -9,20 +9,23 @@ jobs:
   build:
     <<: *defaults
     steps:
+      - run: sudo apt-get install libusb-1.0.0
```

Comment:
note: update circleci config

nothing to be done in this PR, but just a note to myself